.github/workflows/workflow.yml

```diff
@@ -0,0 +1,109 @@
+name: CI/CD Workflow
```

Comment:
This file is based off of e3sm-diags https://github.com/E3SM-Project/e3sm_diags/blob/master/.github/workflows/workflow.yml